.github/workflows/unit.yml

```
@@ -7,10 +7,10 @@ jobs:
     name: unit
     runs-on: ubuntu-22.04
     steps:
-      - uses: actions/setup-go@v3
-        with:
-          go-version: 1.19
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - uses: actions/setup-go@v4
+        with:
+        go-version-file: "go.mod"
```

Indentation here too?
```suggestion
          go-version-file: "go.mod"
```